schema/me/append_conversation_thread.js

```diff
@@ -35,25 +47,27 @@ export default mutationWithClientMutationId({
     },
   },
   outputFields: {
-    message: {
-      type: MessagePayloadType,
-      resolve: message => message,
+    payload: {
+      type: AppendConversationThreadMutationPayload,
+      resolve: data => data,
     },
   },
   mutateAndGetPayload: ({ id, from, to, body_text }, request, { rootValue: { accessToken } }) => {
     if (!accessToken) return null
-    return gravity
-      .with(accessToken, { method: "POST" })("me/token", {
-        client_application_id: IMPULSE_APPLICATION_ID,
-      })
+    let impulseToken
+    return getImpulseToken(accessToken)
       .then(data => {
-        return Promise.resolve(
-          impulse.with(data.token, { method: "POST" })(`conversations/${id}/messages`, {
-            to: ["partner@example.com"],
-            from,
-            body_text,
-          })
-        ).then(message => message)
+        impulseToken = data.token
+        return impulse.with(impulseToken, { method: "POST" })(`conversations/${id}/messages`, {
```

Comment:
Here's the endpoint btw, https://github.com/artsy/impulse/pull/269

The Radiation hookup isnt quite finished yet tho.

We basically need to add `reply_all=true` and can remove the `to`. Then we need `message_id`, which is whatever message is being replied to. For us, it's whatever the last message was. That should be accessible in Emission and can be sent over as a variable in the mutation.